src/features/groups/components/GroupCardList/GroupCard/GroupCardContent/GroupCardDropdownMenuTriggerButton/GroupCardDropdownMenuContent/DeleteGroupDialogTriggerButton/DeleteGroupDialogContent.tsx

```diff
@@ -23,12 +23,17 @@ interface IDeleteGroupDialogContentProps {
    * The function to close this delete dialog.
    */
   onDialogClose: () => void;
+  /**
+   * The function to transition to groupCollection page
+   */
+  onPageTransit?: () => void;
```

Comment:
I add this props which can be undefined because it is not needed if a user is trying to delete a group from group collection page

Reply:
`onXxx` is commonly used for events or functions that are triggered on a state change.

Here is my suggestion to avoid using the keyword `on`:
- `navigate`
- `navigateOnSuccess`

Also, the comment is too specific to explain this property because this component is shared across the project.

Here is my suggestion:
- The function to navigate the user to a different page following a successful deletion.

By the way, the word "navigate" is a more general term when developers talk about page movement. Although "page transition" is also well-understood, the `navigation` describes everything.